content/root/quick-start-guide.textile

```diff
@@ -45,7 +45,9 @@ You're currently viewing the guide in <span lang="javascript">JavaScript</span><
 
 h2(#step-0). Create an Ably account
 
-The following code sample uses a demo API key to authenticate however you can use your own API key if you already have an account, or "sign up":https://ably.com/signup for a free account to obtain one. Just ensure that the key provides the subscribe and publish "capabilities":/core-features/authentication#capabilities-explained.
+"Sign up":https://ably.com/signup for a free account to get your own API key.
+
+The following code sample uses a demo API key to authenticate; however, you can use your API key. Ensure that the key provides the subscribe and publish "capabilities":/core-features/authentication#capabilities-explained.
```

Comment:
> The following code sample uses a demo API key to authenticate; however, you can use your API key. Ensure that the key provides the subscribe and publish "capabilities":/core-features/authentication#capabilities-explained.

Suggested rewording:

The following code sample uses a demo API key to authenticate. Alternatively, if you have an Ably account, you can use your Ably API key. Ensure that your Ably API key includes the subscribe and publish "capabilities":/core-features/authentication#capabilities-explained.